platform/ui/src/components/Icon/getIcon.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+// Icons
+import active from './icons/active.svg';
+import cancel from './icons/cancel.svg';
+import chevronDown from './icons/chevron-down.svg';
+import chevronRight from './icons/chevron-right.svg';
+import infoLink from './icons/info-link.svg';
+import launchArrow from './icons/launch-arrow.svg';
+import launchInfo from './icons/launch-info.svg';
+import settings from './icons/settings.svg';
+import sortingActiveDown from './icons/sorting-active-down.svg';
+import sortingActiveUp from './icons/sorting-active-up.svg';
+import sorting from './icons/sorting.svg';
```

Comment:
Maybe these `svg` files should be placed in another path like `src/assets/icons`, what do you think?

Reply:
I liked it. I will update.